### ISSUE TITLE
User stats: stats bar chart can't be read from the keyboard (or by a screen reader)

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#6322]
## Package
- lib-user

## Describe the bug
The new classification stats bar chart requires a mouse, and the individual bar amounts are only visible on hover. Classifications can't be accessed from the keyboard, which means they can't be accessed in a screenreader either.

Additionally, VoiceOVer announces the currently selected option as `[Object object]` in each of the filter dropdowns.

## To Reproduce
Here's a video where I navigate the user stats page in VoiceOver. Notice how selected menu options aren't announced properly, and the bar chart content is skipped completely.

https://github.com/user-attachments/assets/9ae68cc7-bc77-4114-8e86-1f9bd637b9f4

## Expected behavior
- a volunteer should be able to access their personal classification stats regardless of disability.
- the filter dropdowns should announce the selected option properly.

### Desktop (please complete the following information):
 - OS: MacOS SOnoma 14.6
 - Browser: Safari/VoiceOver and Firefox/VoiceOver

## Additional context
Keyboard access for page content is a level A WCAG requirement ([SC 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html).) Level AA is the commonly accepted level used to define a site as accessible to disabled users.

**Comment by goplayoutside3** [comment]
@eatyourgreens please take a look at https://github.com/zooniverse/front-end-monorepo/issues/5842 and the corresponding Grommet PR that was just accepted. Please add more info to your authored Issue on whether @mcbouslog's Grommet PR is a potential fix or not.

**Comment by eatyourgreens** [comment]
This isn't fixed yet in Grommet. See https://github.com/grommet/grommet/pull/7310#issuecomment-2361275994.

The Zooniverse dev team should know enough about VoiceOver to have made that judgement without asking for my help.

**Comment by eatyourgreens** [comment]
That Grommet PR doesn't fix `[object Object]` in the dropdown menu announcements either, which is a separate bug in the Zooniverse code.

**Comment by eatyourgreens** [comment]
I've suggested a possible fix here: https://github.com/grommet/grommet/issues/7319#issuecomment-2361341210

Again, this a simple thing that the Zooniverse dev team could have picked up by themselves.

**Comment by eatyourgreens** [comment]
I think Mark’s Grommet PR, which just merged, might be using live regions incorrectly. Hence it not working in VoiceOver. 

Hopefully that’s enough info to get you started on debugging it.

**Comment by eatyourgreens** [comment]
Patrick Lauke’s [Why aren’t my live regions working?](https://tetralogical.com/blog/2024/05/01/why-are-my-live-regions-not-working/) is a great resource for understanding how to use HTML5 live regions.

**Comment by eatyourgreens** [comment]
https://github.com/grommet/grommet/pull/7465 is still a bit buggy, because it uses live regions, but it should announce the stats in VoiceOver, when you use the arrow keys to navigate the chart.

**Comment by eatyourgreens** [comment]
These PRs should close this once they're merged:
- #6602.
- grommet/grommet#7465

**Comment by eatyourgreens** [comment]
An update after testing Grommet's data chart details in NVDA: https://github.com/grommet/grommet/pull/7465#issuecomment-2581374802
